.travis.yml

```diff
@@ -6,11 +6,20 @@ branches:
   only:
     - master
 
-go:
-  - 1.11.x
-  - 1.12.x
-  - 1.13.x
-  - tip
+jobs:
+  include:
+    - os: linux
+      go: 1.11.x
+    - os: linux
+      go: 1.12.x
+    - os: linux
+      go: 1.13.x
+    - os: linux
+      go: tip
+    - os: osx
+      go: 1.13.x
```

Comment:
If you want to test just the latest version, you could change this to go: 1.x (see: https://docs.travis-ci.com/user/languages/go/#specifying-a-go-version-to-use).

Reply:
I prefer it being explicit. Thanks for the suggestion though.